src/index.html

```diff
@@ -10,7 +10,51 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form
+      action="big"
+      method="get"
+      class="form search__big"
+      data-qa="big"
+    >
+
+      <img
```

Comment:
```suggestion
    >
      <img
```
Same here. Fix all cases